firestore-send-email/POSTINSTALL.md

```diff
@@ -47,6 +47,7 @@ The top-level fields of the document supply the email sender and recipient infor
 * **ccUids:** An array containing the CC recipient UIDs.
 * **bcc:** A single recipient email address or an array containing multiple recipient email addresses.
 * **bccUids:** An array containing the BCC recipient UIDs.
+* **headers:** An object or array of additional header fields (e.g. `{"X-Key-Name": "key value"}` or `[{key: "X-Key-Name", value: "val1"}, {key: "X-Key-Name", value: "val2"}]`).
```

Comment:
I don't think we want to mirror Nodemailer's messy and unintuitive header input. Let's just do an object with string keys and values unless you have a compelling reason to do otherwise...

Reply:
@mbleigh Can this conversation be resolved now?

Reply:
This probably should have been kept to allow array entry

In emails, it's normal to have duplicate tags with multiple values